### PR TITLE
Build for canary 12

### PR DESCRIPTION
**Pull request by stevemessick**
@devoncarew @pq

### CODE REVIEW

File: product-matrix.json [hunk 1]
```diff
@@ -22,14 +12,14 @@
...
+      "comments": "IntelliJ 2019.1.1, AS 3.5 canary",
       "name": "Android Studio 3.5",
       "version": "3.5",
-      "ijVersion": "2019.1",
+      "ijVersion": "2019.1.1",
       "ideaProduct": "android-studio",
-      "ideaVersion": "191.5416148",
-      "dartPluginVersion": "191.6183.87",
-      "sinceBuild": "191.5416148",
+      "ideaVersion": "191.5471097",
```
Comment (devoncarew) [inline product-matrix.json]: Is rev'ing this intentional (and, will it leave behind users)? We should just verify what version people are using internally (2019.1, or 2019.1.1).

Comment (stevemessick) [reply product-matrix.json]: That's the build artifact, and it is based on 2019.1.

**Comment by stevemessick** [comment]
PTAL. There were some version mis-matches that prevented the plugin from loading in some products.

**Comment by stevemessick** [comment]
TBR, to get the build going.

**Pull request merged**